src/PTY.ts

```diff
@@ -1,32 +1,48 @@
 import * as ChildProcess from "child_process";
 import * as OS from "os";
 import * as _ from "lodash";
-import * as pty from "pty.js";
+import * as pty from "node-pty";
 import {loginShell} from "./utils/Shell";
 import {debug} from "./utils/Common";
 
+let smth = true ? null : pty.createTerminal(undefined, undefined, undefined);
+type ITerminal = typeof smth;
+
 export class PTY {
-    private terminal: pty.Terminal;
+    private terminal: ITerminal;
 
     // TODO: write proper signatures.
     // TODO: use generators.
     // TODO: terminate. https://github.com/atom/atom/blob/v1.0.15/src/task.coffee#L151
     constructor(words: EscapedShellWord[], env: ProcessEnvironment, dimensions: Dimensions, dataHandler: (d: string) => void, exitHandler: (c: number) => void) {
-        const shellArguments = [...loginShell.noConfigSwitches, "-i", "-c", words.join(" ")];
```

Comment:
⚠ Warning: Unix-specific options were replaced by Windows-specific; need to find a portable way to express this option set.